flang/test/Driver/print-resource-dir.F90

```diff
@@ -0,0 +1,3 @@
+! RUN: %flang -print-resource-dir -resource-dir=%S/Inputs/resource_dir \
+! RUN:  | FileCheck -check-prefix=PRINT-RESOURCE-DIR -DFILE=%S/Inputs/resource_dir %s
```

Comment:
You should be able to avoid repeating "%S/Inputs/resource_dir" if you use `define`/`redefine`:
* example: https://github.com/llvm/llvm-project/blob/6d44a1ef55b559e59d725b07ffe1da988b4e5f1c/mlir/test/Integration/Dialect/Linalg/CPU/ArmSVE/matmul.mlir#L1-L14
* docs: https://llvm.org/docs/TestingGuide.html (search for `DEFINE`)

Reply:
I'll make the variable a bit clearer in the next update.  That's a good suggestion, thanks!

As for `ClangExecutable`, I haven't looked at where this is being defined.  It's indeed the driver name and not the name of the Clang driver.  So, for Flang is actually contains the full path to `flang-new`.

I'm not opposed to renaming it as part of this PR, but it may widen the scope and make approval a bit more complex.  So, my feeling is that this refactoring should take place in a different PR.  But I can be convinced otherwise.

Reply:
> So, my feeling is that this refactoring should take place in a different PR.

Makes sense.